exonum-java-binding/fakes/src/main/java/com/exonum/binding/fakes/services/transactions/SetEntryTransaction.java

```diff
@@ -1,89 +0,0 @@
-/*
- * Copyright 2018 The Exonum Team
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.exonum.binding.fakes.services.transactions;
-
-import static com.exonum.binding.common.serialization.StandardSerializers.hash;
-import static com.exonum.binding.common.serialization.StandardSerializers.publicKey;
-import static com.exonum.binding.common.serialization.StandardSerializers.string;
-import static com.google.common.base.Preconditions.checkNotNull;
-
-import com.exonum.binding.common.crypto.PublicKey;
-import com.exonum.binding.common.hash.HashCode;
-import com.exonum.binding.core.storage.database.Fork;
-import com.exonum.binding.core.storage.indices.EntryIndexProxy;
-import com.exonum.binding.core.transaction.Transaction;
-import com.exonum.binding.core.transaction.TransactionContext;
-import com.google.common.annotations.VisibleForTesting;
-
-/**
- * A transaction whose behaviour can be configured. It's not a mock: it writes a given value
- * into the database in its {@link #execute(TransactionContext)}.
- *
- * <p>Such transaction is supposed to be used in TransactionProxy integration tests.
- */
-public final class SetEntryTransaction implements Transaction {
```

Comment:
Why did you remove the whole transaction and not just `info()`?

Reply:
It used to be used in native ITs of Rust TransactionProxy, which are gone. Probably, most of fakes will be gone, but that's after @ipenkin 's tests.